Automattic/wp-calypso
# Remove BlazePress Promote Post i1

Related to SELFSERVE-558

## Proposed Changes
* Since we implemented and successfully delivered i2 of Promoted Posts (BlazePress), we don't need the old code any more. This PR removes both using of the old code and the code itself.

## Testing Instructions



- Run Calypso either locally or using a Live Preview,
- Look through Tools > Advertising and check whether it works fine.

## Pre-merge Checklist
